lib/localenv/clusterenv.go

```diff
@@ -135,7 +135,6 @@ func newClusterEnvironment(args clusterEnvironmentArgs) (*ClusterEnvironment, er
 		Apps:     apps,
 		Users:    users,
 		StateDir: siteDir,
-		Local:    true,
```

Comment:
I factored operation status tailing into a separate function so this is not strictly needed anymore.